profile/README.md

```diff
@@ -1,11 +1,76 @@
+
+## Projects
+
+Since 2023, the primary focus of Nautilus has been on the Torrust Project:
+
+### Torrust
+
+
+`Torrust` is a group of programs that allow for the better management of torrent collections. Nautilus made the commitment to strategically foster this project as it is confident that it is in close alignment with both the Ethos of the company, (and group), and the real company goal nearing itself to a very interesting natural talent pool.
+
+in particular, our ethos is can be expressed as a:
+
+- value of sharing and archival of information. (such as AI and experimental datasets)
+- interest of open networking systems, and standards.
+- indexing and protection of data for future generations.
+
+In addition, for more practical reasons:
+
+- rust is an interesting and increasingly popular programming language
+- there is large communities of passionate users of other torrent trackers and indexes
+
+#### Torrust Tracker
+
+A `Torrent Tracker` is a set of server services that allow for the torrent swam to match-make while optionally keeping useful statistics for better insight and choice making.
+
+The tracker connects peers who share the same torrent (infohash, i.e. a unique specifier for a torrent). A peer that is missing parts or all of the torrent data is then able to connect to other peers, and obtain the parts they are missing.
+
+The torrust tracker is unique as it is a modern implementation in the excellent rust programming language. We hope that this project will become well-used and provide a real benefit to many; and will attract a vibrant community around it's continued improvement and maintenance.
+
+#### Torrust Index
+
+A `Torrent Index` is a software that manages metadata associated, but not necessarily directly embedded inside, torrents. This allows users to find torrents that interest them. A popular example is the "accademictorrents" website. This website provides contextual information about a torrent; such as it's name, upload date, and relevant keywords or category.
+
+Torrent Indexes often all sorts of additional features, such as: user accounts, moderation, a blog, searching and filtering of results, and displaying statistics provided by connected trackers.
+
+Our torrust index is again unique in that it has a very modern design, using the rust programming language, and offers a good range of features.
 
 
-## Who We Are :smiley: :alien: :sunglasses: 🙋‍♀️
 
 We are currently a small team of people working towards our long term vision. Our vision is to create open source software ecosystems, with a particular focus on AI and Machine Learning solutions.
 Our team is both made up of employees and external collaborators. We come from different professional backgrounds and countries.
```

Comment:
I think that it this should come at a different place. Not at the end, and possibly be close to the Ethos section.